c7n/filters/core.py

```diff
@@ -434,8 +434,7 @@ def process_value_type(self, sentinel, value):
                 # EMR not having more functionality.
                 try:
                     value = parse(value, default=datetime.now(tz=tzutc()))
-
-                except (AttributeError, TypeError):
+                except (AttributeError, TypeError, ValueError):
```

Comment:
File "/Users/john_theodore/code/cloud-custodian/lib/python2.7/site-packages/dateutil/parser.py", line 559, in parse
    raise ValueError("Unknown string format")
ValueError: Unknown string format

I'm still getting these. :(